zrcs/zrc-2.md

```diff
@@ -0,0 +1,372 @@
+| @param | `sender`    | `ByStr20` | Address of the sender whose balance is decreased.    |
+| @param | `recipient` | `ByStr20` | Address of the recipient whose balance is increased. |
+| @param | `amount`    | `Uint128` | Amount of tokens to be sent.                         |
+
+|           | Name          | Description                | Event Parameters                                                                                                                                                                                                                   |
+| --------- | ------------- | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| eventName | `OperatorSendSuccess` | Sending is successful.     | `from`: `ByStr20`, `to`: `ByStr20`, and `amount`: `Uint128`.                             |
+| eventName | `Error`       | Sending is not successful. | - emit `CodeNotAuthorised` if the transition is called by a user who is not an approved operator. |
+
+#### 3. Burn
+
+```ocaml
+(* @dev: Burn existing tokens. Only tokenOwner or approved operator can burn a token *)
+(* @param from:                             Address holding the tokens to be burned. *)
+(* @param amount:                           Number of tokens to be destroyed.        *)
+transition Burn(from: ByStr20, amount: Uint128)
+```
+
+|        | Name      | Type      | Description                              |
+| ------ | --------- | --------- | ---------------------------------------- |
+| @param | `from`    | `ByStr20` | Address holding the tokens to be burned. |
+| @param | `amount`  | `Uint128` | Number of tokens to be burned.           |
+
+|           | Name          | Description                | Event Parameters                                                                                                                                                                                                                                               |
+| --------- | ------------- | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| eventName | `BurnSuccess` | Burning is successful.     | `from`: `ByStr20`, and `amount`: `Uint128`.                                                                                                                      |
+| eventName | `Error`       | Burning is not successful. | - emit `CodeNotAuthorised` if the transition is called by a user who is not the contract owner.<br>**NOTE:** Only the `tokenOwner` is allowed to call this transition. |
```

Comment:
There is a bit of confusion here. The first sentence says that only the contract owner can call, but the second one says that the `tokenOwner` should call.

Reply:
Yes, it should say token owner.